headless-server-commerce-parent/pom.xml

```diff
@@ -447,5 +442,37 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <!-- staged cms artifacts in Nexus3-->
+      <id>nexus3-staged-artifacts</id>
+      <repositories>
+        <repository>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+          <id>coremedia-staged-repository</id>
+          <name>CoreMedia Nexus3 Staged repository</name>
+          <url>https://repository.coremedia.com/nexus/repository/staging</url>
+          <layout>default</layout>
+        </repository>
+      </repositories>
+      <pluginRepositories>
+        <pluginRepository>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+          <id>coremedia-staged-repository</id>
+          <name>CoreMedia Nexus3 Staged repository</name>
+          <url>https://repository.coremedia.com/nexus/repository/staging</url>
+          <layout>default</layout>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
```

Comment:
Shouldn't this be held in our local maven settings? Is this something to be used by customers/partners?

Reply:
  👍